atlascope/tests/test_populate.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+
+from atlascope.core.management.commands.populate import command as populate_command
+
+
+@pytest.mark.django_db
+def test_populate_script():
+    populate_command()
```

Comment:
As a general matter, I'm not sure how much this test adds. Running the populate script is a dev task being done all the time, so it's worth thinking about whether adding a test creates more burden on maintenance than just dealing with problems when they come up. In this case, the test is so short that it's probably a wash.

Reply:
True, but on @marySalvi's review of #69, she found several places where I had broken the populate script, and I hadn't realized because I don't run it at the end of developing a feature. Adding a test will ensure that we don't merge breaking changes.